index.md

```diff
@@ -154,6 +154,72 @@ nav[data-toggle='toc'] {
 }
 ```
 
+## Additional options
+
+By default, the plugin finds the first heading level (`<h1>`, then `<h2>`, etc.) that has more than one element and defaults to 1 (for `<h1>`). The identified level becomes the top heading and the plugin supports a single nesting level associated to it, including the subsequent heading only. E.g., if the identified top level is `<h1>`, the nested level will be `<h2>`, which is the next below it; if the top level is `<h2>`, the nested one will be `<h3>`. No additional level to the nested one and no previous level to the top one will be shown.
```

Comment:
Hi, I generally agree about your comments and you can add new commits to this branch. This PR was more than discuss than considering an immediate merge.

> What's the use case here? Do you have an example page where you would want to use this? What would the desired outcome be?

Check this page as an example:

https://ircama.github.io/osm-carto-tutorials/tile-server-ubuntu/

Your plugin is running there.

In fact, at the end of the HTML:

```javascript
  $(function() {
    var navSelector = '#toc';
    var $myNav = $(navSelector);
    Toc.init($myNav, { headings: 'h1,h2,h3,h4,h5,h6' } );
    $('body').scrollspy({
      target: navSelector,
      offset: 220
    });
  });
```

In the page, you can see that my top level is 2 and that I generally use one sublevel, but there are some paragraphs with an additional sublevel 4:

https://ircama.github.io/osm-carto-tutorials/tile-server-ubuntu/#install-mapnik-library-from-package

* [h2] Install Mapnik library
  * [h3] Install Mapnik library from package
  * [h3] Alternatively, install Mapnik from sources
    * [h4] Install Boost from package
    * [h4] Alternatively, install the latest version of Boost from source
    * [h4] Install HarfBuzz from source
    * [h4] Build the Mapnik library from source

You see that h4 items are present (this is thanks to `{ headings: 'h1,h2,h3,h4,h5,h6' }`), are jointly expanded with h3 (and there is no way with this drop to do differently) and they are indented and represented with smaller font size (thanks to the auto-class naming and to the CSS).

> Also, does this differ from `$scope: $('h1,h2,h3,h4,h5,h6')`?

Could you please make an example of this?